### PR TITLE
Update docs to point to 0.7.0

### PR DESCRIPTION
**Pull request by a-roberts**
# Changes

Refer to 0.7.0 in both main readme and install.md

@eddycharly @AlanGreene I know the readme's been vastly improved so lemme know if I've missed anything please. I've uploaded the latest yaml to https://console.cloud.google.com/storage/browser/tekton-releases/dashboard/latest/ as well.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,13 +28,16 @@ Among other things, the Tekton Dashboard supports:
...
+It is **strongly recommended** to use the **v0.6.1.4** release or newer for Tekton Pipelines v0.11.x and v0.12.x and Tekton Triggers v0.4.
 - This is a **critically important security release**
 - Earlier versions are deprecated and should be used for **development or isolated usage only**
 
+The latest release is **v0.7.0**.
+
 | Version | Docs | Pipelines | Triggers |
 | ------- | ---- | --------- | -------- |
 | [HEAD](./DEVELOPMENT.md) | [Docs @ HEAD](./docs/README.md) | v0.11.x - v0.13.x | v0.5.x |
+| [v0.7.0](https://github.com/tektoncd/dashboard/releases/tag/v0.7.0) | [Docs @ v0.7.0](https://github.com/tektoncd/dashboard/tree/v0.7.0/docs) | v0.11.x - v0.13.x | v0.4.x -> 0.5.x |
```
Comment (eddycharly) [inline README.md:40]: Looks good.
Any reason to use an arrow here ? (`->`)

Comment (a-roberts) [reply README.md]: None, fixed

File: README.md [hunk 2]
```diff
@@ -28,13 +28,16 @@ Among other things, the Tekton Dashboard supports:
 
 ## Read the docs
 
-It is **strongly recommended** to use the **v0.6.1.4** release or newer for Tekton Pipelines v0.11.x and v0.12.x and Tekton Triggers v0.4
+It is **strongly recommended** to use the **v0.6.1.4** release or newer for Tekton Pipelines v0.11.x and v0.12.x and Tekton Triggers v0.4.
 - This is a **critically important security release**
 - Earlier versions are deprecated and should be used for **development or isolated usage only**
 
+The latest release is **v0.7.0**.
```
Comment (AlanGreene) [inline README.md:35]: Do we need to explicitly call this out or is it obvious enough from the table below?

Comment (a-roberts) [reply README.md]: Gone, obvious enough

**Pull request merged**